src/pykiso/lib/auxiliaries/udsaux/common/uds_base_auxiliary.py

```diff
@@ -0,0 +1,146 @@
+##########################################################################
+# Copyright (c) 2010-2022 Robert Bosch GmbH
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
+
+"""
+uds_base_auxiliary
+******************
+
+:module: uds_base_auxiliary
+
+:synopsis: Auxiliary base class providing common methods for
+    Unified Diagnostic Service protocol handling.
+    This base auxiliary is not meant to be used directly, use its
+    subclasses instead.
+
+.. currentmodule:: uds_base_auxiliary
+
+"""
+import configparser
+import logging
+from pathlib import Path
+from typing import Optional, Union
+
+from uds import Uds, createUdsConnection
+
+from pykiso.connector import CChannel
+from pykiso.interfaces.thread_auxiliary import AuxiliaryInterface
+
+log = logging.getLogger(__name__)
+
+
+class UdsBaseAuxiliary(AuxiliaryInterface):
+    """Base Auxiliary class for handling UDS protocol."""
+
+    POSITIVE_RESPONSE_OFFSET = 0x40
+
+    def __init__(
+        self,
+        com: CChannel,
+        config_ini_path: Union[Path, str],
+        odx_file_path: Optional[Union[Path, str]] = None,
+        request_id: Optional[int] = None,
+        response_id: Optional[int] = None,
+        **kwargs,
+    ):
+        """Initialize attributes.
+
+        :param com: communication channel connector.
```

Comment:
use same order as defined in the init method

Reply:
Fixed